frontend/src/app/sessions/session/tiling-window-manager/tiling-window-manager.component.ts

```diff
@@ -0,0 +1,199 @@
+/*
+ * SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Component, HostListener, Input, OnInit } from '@angular/core';
+import { Session } from 'src/app/schemes';
+import { FullscreenService } from '../../service/fullscreen.service';
+
+@Component({
+  selector: 'app-tiling-window-manager',
+  templateUrl: './tiling-window-manager.component.html',
+  styleUrls: ['./tiling-window-manager.component.css'],
+})
+export class TilingWindowManagerComponent implements OnInit {
+  private _tilingSessions: TilingSession[] = [];
+
+  @Input()
+  set sessions(sessions: Session[]) {
+    this._tilingSessions = sessions.map((session, index) => ({
+      ...session,
+      index: index,
+      width: -1,
+      fullscreen: false,
+    }));
+    this.resetWidths();
+  }
+
+  get sessions(): TilingSession[] {
+    return this._tilingSessions.sort((a, b) => a.index - b.index);
+  }
+
+  public resizeState: ResizeState = {};
+
+  private minimumSessionWidth = 0;
+  private existingMargin = 0;
+
+  constructor(public fullscreenService: FullscreenService) {}
+
+  ngOnInit(): void {
+    this.fullscreenService.isFullscreen$.subscribe((isFullscreen) => {
+      this.existingMargin = isFullscreen ? 0 : 27.48;
```

Comment:
Where does the margin of `27.48` come from?

Reply:
From inspection to my browser (left and right margins when not being in full screen mode). But I agree that it's not a good idea to just put it as it's there. However, I have tested whether or not having it makes a difference that doesn't. So I just remove the entire `existingMargin`.